pandas/io/_plugin_loader.py

```diff
@@ -51,7 +51,9 @@ def writer(self, fname, mode='w'):
 make an arbitrary decision on which to use.
 """
 import functools
+import warnings
 from importlib.metadata import entry_points
+import importlib_metadata
```

Comment:
Is this an external package? Then make it an (optional) dependency?

Reply:
It's a standard library I think, I'll confirm, in case it was installed in the environment by some other package.

Reply:
looks like the stdlib one is importlib.metadata`, and `importlib_metadata` is the backport - does it work with the stdlib one? seems it's new in py3.8 https://docs.python.org/3/library/importlib.metadata.html